models/place.json

```diff
@@ -272,5 +160,107 @@
         "type": "Available"
       }
     }
+  ],
+  "properties": {
+    "@id": {
+      "$ref": "./place-@id.json"
+    },
+    "mainLanguage": {
+      "$ref": "./place-mainLanguage.json"
+    },
+    "name": {
+      "$ref": "./place-name.json"
+    },
+    "terms": {
+      "$ref": "./place-terms.json"
+    },
+    "calendarType": {
+      "$ref": "./place-calendarType.json"
+    },
+    "address": {
+      "$ref": "./place-address.json"
+    },
+    "geo": {
+      "$ref": "./place-geo.json"
+    },
+    "startDate": {
+      "$ref": "./place-startDate.json"
+    },
+    "endDate": {
+      "$ref": "./place-endDate.json"
+    },
+    "status": {
+      "$ref": "./place-status.json"
+    },
+    "openingHours": {
+      "$ref": "./place-openingHours.json"
+    },
+    "availableFrom": {
+      "$ref": "./place-availableFrom.json"
+    },
+    "availableTo": {
+      "$ref": "./place-availableTo.json"
+    },
+    "sameAs": {
+      "$ref": "./place-sameAs.json"
+    },
+    "created": {
+      "$ref": "./place-created.json"
+    },
+    "modified": {
+      "$ref": "./place-modified.json"
+    },
+    "creator": {
+      "$ref": "./place-creator.json"
+    },
+    "workflowStatus": {
+      "$ref": "./place-workflowStatus.json"
+    },
+    "languages": {
+      "$ref": "./place-languages.json"
+    },
+    "completedLanguages": {
+      "$ref": "./place-completedLanguages.json"
+    },
+    "typicalAgeRange": {
+      "$ref": "./place-typicalAgeRange.json"
+    },
+    "description": {
+      "$ref": "./place-description.json"
+    },
+    "priceInfo": {
+      "$ref": "./place-priceInfo.json"
+    },
+    "contactPoint": {
+      "$ref": "./place-contactPoint.json"
+    },
+    "bookingInfo": {
+      "$ref": "./place-bookingInfo.json"
+    },
+    "mediaObject": {
+      "$ref": "./place-mediaObject.json"
+    },
+    "image": {
+      "$ref": "./place-image.json"
+    },
+    "videos": {
+      "$ref": "./place-videos.json"
+    },
+    "labels": {
+      "$ref": "./place-labels.json"
+    },
```

Comment:
I think the validation of labels is currently incorrect in the schema. It's set to 1-50 characters instead of 1-255, and the regex also seems wrong compared to organizers and regex in the code.

Reply:
(Same for `hiddenLabels`)